README.md

```diff
@@ -5,12 +5,22 @@ eslint config rules for Particle Javascript projects
 
 1. Install dependencies
 `npm install --save-dev eslint eslint-config-particle`
-1. Tell eslint to use the Particle config by creating `.eslintrc.js`
+1. Tell eslint to use the Particle config by creating `eslint.config.mjs`
 ```
-module.exports = {
-  extends: ['eslint-config-particle'],
-  root: true
-}
+import eslintParticle from 'eslint-config-particle';
```

Comment:
Ah... I missed that ESLint 9.x is Node 18.18 and higher, which is why it expects that `structuredClone` to be defined, node 17 was the first version with that.

Reply:
On device-service, using Node.js 18, I got 3000 or so lint errors, the vast majority were the mocha stuff insisting on using regular functions instead of arrow functions, so you can `this.` instead of working with globals, I can disable that option.

The next most common problems are JSDoc related, and most of them I agree with, because it is JSDoc without types or mismatched names. Some of the type name problems get fixed with `--fix`

If I disable the JSDoc checker and run `--fix`, I get about 300 remaining problems, most common of which are line length or doing setup in describe functions which is discouraged by the mocha plugin, because it favors `this.` style and having the `before()` function take care of adding that stuff to `this`.

It feels like a tractable set of changes in that repository at least.